src/analyze/analyze.c

```diff
@@ -1081,18 +1080,50 @@ static int analyze_blame(int argc, char *argv[], void *userdata) {
         if (n <= 0)
                 return n;
 
-        typesafe_qsort(times, n, compare_unit_time);
+        table = table_new("TIME", "UNIT");
```

Comment:
we should probably stick to lowercase table header fields, as the formatter uppercases them anyway, and this way the JSON output of the formatter is nicer as it uses the table header fields 1:1 for the json output.

also see:

https://github.com/systemd/systemd/pull/13075

Reply:
Let's do this in a later patch. There's already code in src/network that uses uppercase, so we should fix them all in one swell swoop.

I noticed some minor issues with formatting when reviewing this PR, but they predate the PR. I'll submit fixes separately.